gateway/api_definition.go

```diff
@@ -494,7 +494,7 @@ func (a APIDefinitionLoader) FromDashboardService(endpoint string) ([]*APISpec,
 	}
 
 	// Extract tagged APIs#
-	list := &model.MergedAPIList{}
+	list := model.NewMergedAPIList()
```

Comment:
do these changes needed to be in this PR?

Reply:
updates internal api usage, so ideally yes